diagnostic.md

```diff
@@ -5,74 +5,77 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+The back end, or server side, is code that remains on the server.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```bash
-// your response here
+Server
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```bash
-// your response here
+Controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```bash
-// your response here
+Controller handles events from the view.
 ```
```

Comment:
Because SPA methodology and more robust front end tools are better than the old view standard of traditional MVC's like Rails.